server.js

```diff
@@ -1,49 +1,45 @@
-let categories = require('./my-app-react/src/data/categories.json')
-let products = require('./my-app-react/src/data/products.json')
+let dataService = require('./dataservice.js');
 
-const express = require('express')
-const app = express()
+const express = require('express');
+const app = express();
 
 app.get('/info', (req, res) => {
-    info = [
-        {"serverName": "localhost5000"},
+    let info = [
+        {"serverName": "localhost5001"},
         {"sampleName": "chinemerem-react"},
         {"serverVersion": "1.0.0"}
-    ]
+    ];
     res.json(info)
-})
+});
 
 app.get('/products/all', (req, res) => {
-    res.json(products)
-})
+    let productsMap = dataService.getCombinedProductMap();
+    let allProducts = [];
+    productsMap.forEach(value => {
+       allProducts.push(value)
+    });
```

Comment:
No need for [these lines](https://github.com/felixfavour/my-react-app/pull/10/files#diff-78c12f5adc1848d13b1c6f07055d996eR17-R20), conversion not required. Simply return the map.

Reply:
apparently, A map is rendered in the page as `{ }`